cg/meta/deliver.py

```diff
@@ -175,7 +175,8 @@ def deliver_sample_files(
         if not self.dry_run:
             delivery_base.mkdir(parents=True, exist_ok=True)
         file_path: Path
-        number_linked_files: int = 0
+        number_currently_linked_files: int = 0
```

Comment:
I think this name is misleading, prefer number_of_files_linked_now

Reply:
For all we know, the number of files linked currently is 100. But maybe there is a better name

Reply:
true!